bluemira/geometry/tools.py

```diff
@@ -757,7 +757,7 @@ def sweep_shape(profiles, path, solid=True, frenet=True, label=""):
     return convert(result, label=label)
 
 
-def distance_to(geo1: BluemiraGeo, geo2: BluemiraGeo):
+def distance_to(geo1: Union[Iterable, BluemiraGeo], geo2: Union[Iterable, BluemiraGeo]):
```

Comment:
Should this be `Union[Iterable[BluemiraGeo], BluemiraGeo]`?

Also, this function's docstring needs updating too.